addon/validations/factory.js

```diff
@@ -633,6 +633,8 @@ function getCPDependentKeysFor(attribute, model, validations) {
     dependentKeys.push('model.isDeleted');
   }
 
+  dependentKeys = dependentKeys.filter(dependentKey => !!dependentKey);
```

Comment:
This can be combined with the map below.

```js
dependentKeys = dependentKeys.filter(Boolean).map(...)
```

Reply:
Do you mean as: 
```
dependentKeys = dependentKeys
    .filter(dependentKey => !!dependentKey)
    .map(d => {
      return d.replace(/^model\./, `${ATTRS_MODEL}.`);
    });
```

Reply:
```js
dependentKeys = dependentKeys
    .filter(Boolean)
    .map(d => d.replace(/^model\./, `${ATTRS_MODEL}.`));
```

Reply:
Ah. Got it. Done!